webpack/components/extensions/HostDetails/Tabs/PackagesTab.js

```diff
@@ -183,16 +202,20 @@ export const PackagesTab = () => {
             toggleGroup,
           }
           }
-          additionalListeners={[hostId, packageStatusSelected]}
+          additionalListeners={[hostId, packageStatusSelected,
+            activeSortDirection, activeSortColumn]}
           fetchItems={fetchItems}
           autocompleteEndpoint={`/hosts/${hostId}/packages/auto_complete_search`}
           foremanApiAutoComplete
           variant={TableVariant.compact}
         >
           <Thead>
             <Tr>
-              {columnHeaders.map(col =>
-                <Th key={col}>{col}</Th>)}
+              <SortableColumnHeaders
+                columnHeaders={columnHeaders}
+                pfSortParams={pfSortParams}
+                columnsToSortParams={COLUMNS_TO_SORT_PARAMS}
+              />
               <Th />
```

Comment:
Just noticed this (not your code), but is there ever a permission that should disable this column.

Reply:
Not a permission but I think column customization has been requested in the past.  But we can cross that bridge when we get there :stuck_out_tongue_winking_eye: